src/main/java/fi/aalto/cs/apluscourses/model/Points.java

```diff
@@ -60,6 +60,10 @@ public Points(@NotNull Map<Long, List<Long>> exercises,
     return exercises.getOrDefault(exerciseGroupId, Collections.emptyList());
   }
 
+  public int getExercisesAmount() {
+    return exercises.values().stream().mapToInt(List::size).sum();
+  }
+
```

Comment:
Should this be calculated only once (either in constructor or lazily when first needed)?

Reply:
sure